traveler-swift-core/TravelerKitTests/Mocking/Endpoints.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  Endpoints.swift
+//  TravelerKitTests
+//
+//  Created by Omar Padierna on 2019-10-26.
+//  Copyright © 2019 Ata Namvari. All rights reserved.
+//
+
+import Foundation
+
+enum APISection {
+    case authenticate(key: String)
+    case flights(number: String, date: Date)
+    case catalog(flights: [String]?)
+}
+
+struct Endpoints {
+
+    let baseURLs = [URL(string: "https://traveler.dev.guestlogix.io")!, URL(string: "https://traveler.rc.guestlogix.io")!, URL(string: "https://traveler.guestlogix.io")!]
```

Comment:
Suggestion: Rather than making a struct for the endpoints, make a computed property in the enum that contains the urls.  Usage would be: `{APISection}.urls` for example.